daffodil-cli/src/test/scala/org/apache/daffodil/cli/cliTest/TestCLItdml.scala

```diff
@@ -110,6 +110,16 @@ class TestCLItdml {
     }(ExitCode.Success)
   }
 
+  @Test def test_CLI_Tdml_Trace_singleTest(): Unit = {
+    val tdml = path(
+      "daffodil-test/src/test/resources/org/apache/daffodil/section06/entities/Entities.tdml",
+    )
+
+    runCLI(args"-t test $tdml byte_entities_6_08") { cli =>
+      cli.expect("[Pass] byte_entities_6_08")
```

Comment:
Please add comments here about what you'd like to test, but how expect isn't letting you do so. 

Also add a JIRA ticket about that issue and reference it from the comments.

Reply:
Done